src/constants/common/page.ts

```diff
@@ -4,6 +4,7 @@ export const PAGE_PATH = {
   matching: {
     index: '/matching',
     loading: '/matching/loading',
+    playing: '/matching/playing',
```

Comment:
p3;
페이지 명을 좀 고민해봤는데요.
"랜덤 매칭 연결 중"과 "loading" 의미에 차이가 있다고 느껴집니다.
그리고 playing도 조금 애매하게 느껴져서 다음과 같은 명칭 어떤지 조심스레 의견 내봅니다!

```suggestion
    connecting: '/matching/connecting',
    matchUp: '/matching/matchUp',
```

추가적으로 혹시 page url일 경우에는 네이밍을 카멜 케이스 대신 케밥 케이스(`-` 사용) 사용했으면 좋겠는데 다른 분들 의견이 궁금해요!
예시: `match-up`, `find-password`

@sohyeonAn 소현님 page url에 대한 의견 부탁드려요!

Reply:
앗앗 맞습니다 저도 생각하고 있었던 부분이에요. 

제가 비밀번호 찾기 페이지를 만들 때 생각없이(?) 카멜 케이스를 사용했었는데 이후에 찾아보니 URL에는 케밥 케이스를 사용하는 게 일반적인 것 같더라구요.

제가 작업한 비밀번호 관련 페이지도 이후 작업하면서 케밥 케이스로 변경해 두도록 하겠습니다!

Reply:
저도 코딩하면서 자꾸 loading이라는 명이 걸렸었습니다! 확인 감사합니다!
해당 PR이 조금 크다고 생각하여, 페이지(컴포넌트)명은 별도의 이슈에서 작업하도록하겠습니다!

Reply:
작업 완료했습니다!

작업 커밋: 98b4473